anyblok_furetui/furetui/tests/test_furetui.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from ...testing import TmpTemplate
+
+
+tmpl_test_form_ok = """
+    <template id="tmpl_test_form">
+        <field name="name" />
+    </template>
+"""
+
+
+tmpl_test_form_ko = """
+    <template id="tmpl_test_form">
+        <button label="test" />
+        <field name="name" />
+    </template>
+"""
+
+
+tmpl_test_list_ok = """
+    <template id="tmpl_test_list">
+        <field name="name" />
+    </template>
+"""
+
+
+tmpl_test_list_ko = """
+    <template id="tmpl_test_list">
+        <buttons>
+            <button label="test" />
+        </buttons>
+        <field name="name" />
+    </template>
+"""
+
+
+@pytest.mark.usefixtures('rollback_registry')
+class TestResourceForm:
```

Comment:
for KO errors it could be great to find a way to make sure it's the expected exception, I believe message is something like self.identity is not present on blablabla

Reply:
I understand, maybe later :/